### ISSUE TITLE
Npm release 3.8.0 is missing some type definitions

### ISSUE THREAD

**Issue by rchl** [microsoft/vscode-json-languageservice#66]
Trying to compile `vscode-json-languagesserver` (not "service") results in an error because npm-released version of this package is missing so type definitinions.

```
node_modules/vscode-json-languageservice/lib/umd/jsonLanguageService.d.ts:7:35 - error TS7016: Could not find a declaration file for module './parser/jsonParser'. '/Users/projects/vscode-json-languageserver/node_modules/vscode-json-languageservice/lib/umd/parser/jsonParser.js' implicitly has an 'any' type.

7 export { IApplicableSchema } from './parser/jsonParser';
                                    ~~~~~~~~~~~~~~~~~~~~~


Found 1 error.
```

There are only definitions for the files in the root of `umd` directory but not in `./parser` and such:
![Screenshot 2020-08-15 at 09 11 46](https://user-images.githubusercontent.com/153197/90307535-5a260f00-ded7-11ea-964a-c49cfd0b9d1c.png)

**Comment by aeschli** [comment]
The import was unnecessary. Published 3.8.1 with the fix.

**Comment by rchl** [comment]
Works now. Thanks. 👍
